tasmota/tasmota_configurations.h

```diff
@@ -406,6 +406,264 @@
+#define USE_DOMOTICZ                             // Enable Domoticz (+6k code, +0.3k mem)
+//#define USE_HOME_ASSISTANT                       // Enable Home Assistant Discovery Support (+7k code)
+
+//#define USE_MQTT_TLS                             // Use TLS for MQTT connection (+34.5k code, +7.0k mem and +4.8k additional during connection handshake)
+//  #define USE_MQTT_TLS_CA_CERT                   // Force full CA validation instead of fingerprints, slower, but simpler to use (+2.2k code, +1.9k mem during connection handshake)
+//  #define USE_MQTT_TLS_FORCE_EC_CIPHER           // Force Elliptic Curve cipher (higher security) required by some servers (automatically enabled with USE_MQTT_AWS_IOT) (+11.4k code, +0.4k mem)
+//  #define USE_MQTT_AWS_IOT                       // Enable MQTT for AWS IoT - requires a private key (+11.9k code, +0.4k mem)
+
+//#define USE_KNX                                  // Enable KNX IP Protocol Support (+9.4k code, +3k7 mem)
+#define USE_WEBSERVER                            // Enable web server and Wifi Manager (+66k code, +8k mem)
+  #define USE_JAVASCRIPT_ES6                     // Enable ECMAScript6 syntax using less JavaScript code bytes (fails on IE11)
+#define USE_WEBSEND_RESPONSE                   // Enable command WebSend response message (+1k code)
+#define USE_EMULATION_HUE                      // Enable Hue Bridge emulation for Alexa (+14k code, +2k mem)
+//  #define USE_EMULATION_WEMO                     // Enable Belkin WeMo emulation for Alexa (+6k code, +2k mem)
+//  #define USE_SENDMAIL
+
+//#define USE_DISCOVERY                            // Enable mDNS for the following services (+8k code, +0.3k mem)
+#define USE_TIMERS                               // Add support for up to 16 timers (+2k2 code)
+  #define USE_TIMERS_WEB                         // Add timer webpage support (+4k5 code)
+  #define USE_SUNRISE                            // Add support for Sunrise and sunset tools (+16k)
+
+// -- Compression ---------------------------------
+#define USE_UNISHOX_COMPRESSION                  // Add support for string compression in Rules or Scripts
+
+#define USE_RULES                                // Add support for rules (+4k4 code)
+//  #define USE_EXPRESSION                         // Add support for expression evaluation in rules (+3k2 code, +64 bytes mem)
+//    #define SUPPORT_IF_STATEMENT                 // Add support for IF statement in rules (+4k2 code, -332 bytes mem)
+//  #define SUPPORT_MQTT_EVENT                     // Support trigger event with MQTT subscriptions (+3k5 code)
+
+//#define USE_SCRIPT                               // Add support for script
+  #define USE_SCRIPT_FATFS 4                     // Add support for script storage on SD card (+12k code, +4k mem)
+
+// -- Optional modules ----------------------------
+//#define ROTARY_V1                                // Add support for MI Desk Lamp
```

Comment:
As this is enabled in my_user_cfg.h shouldn't they all be undef?

Reply:
right, changed